features/246-custom-domain/index.md

```diff
@@ -80,6 +81,22 @@ Allow developer to setup custom domain for their application, basically followin
 - If the provided `A` record is not present, the custom domain would not be accessible directly. Instead, developer should configure reverse proxy (e.g. CDN) to direct traffic from custom domain to app default domain. The gateway would verify the forwarded host (i.e. custom domain) is valid.
 - Let's Encrypt HTTP certificate would be issued only if the provided `A` record is present.
 
+## Domain Assignment
+
+Custom domain can be assigned to microservices or gears. By default, custom
+domains are assigned to microservices. Developers can change assignment by
+updating custom domain using `skycli`.
+
+For example, suppose developer would like to have 2 custom domains:
+- `my-app.com`: serve main app (i.e. microservice)
+- `accounts.my-app.com`: serve auth service (i.e. auth gear)
+
+Developer can add `my-app.com` and `accounts.my-app.com` as usual, then use
+skycli to assign `accounts.my-app.com` to auth gear:
+```
+skycli domain update accounts.my-app.com --assign-to=auth
```

Comment:
After user setup custom domain to microservices, should we show some warning or hints that they should assign domain to auth too otherwise they can not use cookie as session transport (first party app case).